java-client/src/generated/java/org/opensearch/client/opensearch/_types/DocStatus.java

```diff
@@ -66,7 +66,7 @@ public class DocStatus implements PlainJsonSerializable, ToCopyableBuilder<DocSt
     private final Integer _1xx;
 
     @Nullable
-    private final Integer _2xx;
+    private final Long _2xx;
```

Comment:
Nice catch, thank you @donggyu04 , could you please update all counters to be `Long` (to reflect server side APIs), thank you

Reply:
@reta 

Hi~ I have created a [PR](https://github.com/opensearch-project/opensearch-api-specification/pull/933) to change the field type in the `opensearch-api-specification` repository :)